realm/realm-library/src/androidTest/java/io/realm/RealmTests.java

```diff
@@ -781,6 +781,7 @@ public void realmConfiguration_fileName() {
     }
 
     @Test
+    @Ignore
```

Comment:
Sorry I wasn't clear. You are disabling this test since it will take too long, right? Is there  a way where we can run it?